### DISCUSSION TITLE
Swiper Slides permanently changing width

### DISCUSSION THREAD

**Discussion by marco910** [nolimits4web/swiper#5931] [Q&A]
I'm using Swiper version 8.0.7 in a React project and having some trouble with the slider in Firefox (macOS, Firefox version 103.0).

If I open up the page where Swiper is used in Firefox the width of each slide is permanently changing. This only happens in Firefox and no other browser. There are no console errors or anything else which gives and idea of what goes wrong.
Here's a screen recording of the issue:

https://user-images.githubusercontent.com/57726854/182184295-c26b59db-9ad1-4040-ab90-61b7b38ed687.mov

Any ideas how I can fix this?

**Comment by winley** [comment]
Did anyone get an answer with this - have tried differenet version and it seems unless its called .swiper it will not load correctly and I get exactly this issue.

**Reply by marco910** [reply]
@winley After testing a few hours around I found out that it was caused by the CSS grid of the parent element on my site. The Swiper was nested inside a container, which was a CSS grid with columns that used `fr`. 
Hope this helps.

**Comment by charlie-george-1989** [comment]
I'm getting the same behaviour with swiper-slide width and translate attributes.

mean-ice.localsite.io

**Comment by Yuniac** [comment]
Same here... only on firefox, no idea what's going on but I have to use chrome to finish this feature I guess

**Reply by charlie-george-1989** [reply]
Let us know if you find a fix!
